packages/rich-text-editor/src/RichTextEditor/components/ToolbarItem/ToolbarItem.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import styles from "./ToolbarItem.scss"
+
+export interface ToolbarItemProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode
+}
+
+export const ToolbarItem: React.VFC<ToolbarItemProps> = props => (
```

Comment:
Looks like you forgot to remove all of the ToolbarItem stuff given you're not using it anymore